dev/package-examples/auditd-2.0.4/dataset/auditd/agent/stream/stream.yml

```diff
@@ -0,0 +1,49 @@
+type: auditd/auditd
```

Comment:
@ph What is the type for auditbeat modules?

Reply:
@ruflin https://docs.google.com/spreadsheets/d/1jv_zQIlhv88BYXcpr3i2mo2MVB0OpPUPMYqH8JeSUjU/edit#gid=1351342325

In the above case audit/auditd

Reply:
thanks, adjusted.